### ISSUE TITLE
Issue when uploading tour and heading set to true

### ISSUE THREAD

**Issue by himynamesdave** [trekview/tourer#26]
```
./trekview.py createtour /Users/davidgreenwood/Downloads/UKFP003
Directory: /Users/davidgreenwood/Downloads/UKFP003
Do you agree to Google’s Terms of Service? https://policies.google.com/terms [y/N]: y
Validating files before upload
Do you want to connect photos in ascending time order (recommended) [T], ascending alphabetical order [A], or no connection to be made [N]? If no selected, Google will automatically attempt to link your photos if they are no more than 5 meters apart: t
Do you want to Tourer to calculate photo heading (direction)? This is recommended if the connected photos have a clear line of sight between them. [Y]/N [Y]: y
Please enter a new tour name: Fleet Pond test
Please select tour type: 
1. Land-Drive
2. Land-Hike
3. Land-Bike
4. Land-Climb
5. Land-Ski
6. Land-Snowboard
7. Land-Skateboard
8. Land-Rollerblade
9. Land-OtherLand
10. Water-Sail
11. Water-Kayak
12. Water-Raft
13. Water-StandupPaddleBoard
14. Water-OtherWater
15. Air-Drone
16. Air-HangGlide
17. Air-Parachute
18. Air-Windsuit
19. Air-Plane
20. Air-OtherAir
: 2
Please enter tour description: Just a description
Please enter tour tags, comma-separated: pond, water, something else
New tour saved, tour ID: 1
Ready to upload 10 file(s) for tour Fleet Pond test, ID 1
Starting upload
  0%|                                                    | 0/10 [00:00<?, ?it/s]Traceback (most recent call last):
  File "./trekview.py", line 896, in <module>
    cli()
  File "/Users/davidgreenwood/.pyenv/versions/3.6.5/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/davidgreenwood/.pyenv/versions/3.6.5/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/davidgreenwood/.pyenv/versions/3.6.5/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/davidgreenwood/.pyenv/versions/3.6.5/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/davidgreenwood/.pyenv/versions/3.6.5/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "./trekview.py", line 893, in createtour
    upload_files(path)
  File "./trekview.py", line 499, in upload_files
    "heading": float(newphoto.heading),
TypeError: float() argument must be a string or a number, not 'NoneType'
```

Trying to upload linked panoramas as tour and get above error.

Seems to be issue with defining heading.

**Comment by himynamesdave** [comment]
https://drive.google.com/file/d/170eOXcmbWPGT_BjWvjmjZtYvfD5QN8EN/view?usp=sharing

**Comment by annndrey** [comment]
Fixed in bd8562918faa3490bca0145b3cc91a4a4b9bb73d
